src/hydrusvideodeduplicator/dedup.py

```diff
@@ -311,8 +342,9 @@ def _find_potential_duplicates(
                                     for video2_hash in islice(hashdb, row["farthest_search_index"], None)
                                 )
 
-                                # Update furthest search position to the current length of the table
-                                row["farthest_search_index"] = len(hashdb) - 1
+                                # Video has now been compared against all other videos for dupes, so update
+                                # farthest_search_index to the current length of the table
+                                row["farthest_search_index"] = len(hashdb)
```

Comment:
@ianwal This change brings setting the index here in line with how we set it earlier in the file, via ` row.setdefault("farthest_search_index", i + 1)`. I believe how it used to be set was making us compare an extra file that had already been checked sometimes.

Reply:
Good, I think this is right